src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -615,6 +604,7 @@ static void addERR_ParamsCantBeWithModifier(BindingDiagnosticBag diagnostics, Sy
             Symbol? owner,
             BaseParameterSyntax syntax,
             int ordinal,
+            int parameterCount,
```

Comment:
Rather than `parameterCount`, and adding/subtracing 1 on either side, consider doing `lastParameterIndex` and using the number directly.

Reply:
Thanks, changed: 1d9eca76a799386203554d0cd0e772c8a68a933a